strongbox-security/strongbox-authentication-registry/src/test/java/org/carlspring/strongbox/authentication/ConfigurableProviderManagerTest.java

```diff
@@ -76,22 +73,23 @@
     public void testExternalUserCache()
     {
         //Check that we have proper implementation of UserDetailsService injected by Spring
-        assertTrue(userDetailsService instanceof ConfigurableProviderManager);
+        assertThat(userDetailsService instanceof ConfigurableProviderManager).isTrue();
```

Comment:
`assertThat(userDetailsService instanceof ConfigurableProviderManager).isTrue();`

=>

`assertThat(userDetailsService).isInstanceof(ConfigurableProviderManager.class)`

please change it everywhere where `instanceof` is in use in tests